Vježba6b

dbZrg opened the pull request:
Darij Baldasari

dbZrg committed:
Darij Baldasari - Mislim da je gotovo, osim ako treba omogućiti promjenu boje fonta

ncaklovic commented:
Uglavnom radi, ali par sitnica za popraviti:
- font dialog nije modalni
- font se promijeni i na "Cancel" na font dialogu
- nisam siguran da želite CF_EFFECTS kontrole ako ne podržavate boju teksta
- umjesto (nesigurnog) sprintf-a radije koristite [std::to_string](https://en.cppreference.com/w/cpp/string/basic_string/to_string)

dbZrg committed:
-font dialog je sada modalan
-dodao LOGFONT strukturu (tempFont) kako bih izbjegao promjenu lfont strukture, lfont se sada izmjeni jedino ako ChooseFont vrati TRUE tj. kliknut je "OK" na dijalogu
-maknuo CF_EFFECTS
-zamjenio sprintf sa std::to_string

ncaklovic commented:
Sve u redu.
